packages/openapi-metadata/build.config.ts

```diff
@@ -0,0 +1,20 @@
+import { defineBuildConfig } from "unbuild";
+
+export default defineBuildConfig({
+  entries: [
+    "./src/index.ts",
+    "./src/decorators/index.ts",
+    "./src/metadata/index.ts",
+    "./src/errors/index.ts",
+    "./src/ui/index.ts",
+  ],
+  declaration: "compatible",
+  clean: true,
+  sourcemap: true,
+  rollup: {
+    // Ship CommonJS-compatible bundle
+    emitCJS: true,
+    // Don’t bundle .js files together to more closely match old exports (can remove in next major)
+    output: { preserveModules: true },
```

Comment:
openapi-metadata was already using unbuild! But adding this config just shores it up more closely with the other packages without changing output too much.